templates/sso-cd-image-stream.json

```diff
@@ -45,21 +45,21 @@
                         }
                     },
                     {
-                        "name": "1.0",
```

Comment:
@martin-kanis Please keep the _1.0_ image tag here (for backward-compatibility), but add completely new _5.0_ record (basically copy everything between the curly braces, and change the name to _5.0_).